[docs] Fixed -s option desc of srt-live-transmit (#1247)
Haivision/srt: docs/srt-live-transmit.md

- **-verbose, -v** - Display additional information on the standard output. Note that it's not allowed to be combined with output specified as **file://con**.
- **-statsout** - SRT statistics output: filename. Without this option specified, the statistics will be printed to the standard output.
- **-pf**, **-statspf** - SRT statistics print format. Values: json, csv, default.

> - **-s**, **-stats**, **-stats-report-frequency** - The frequency of SRT statistics collection, in the number of packets.

Reviewer: ```suggestion
- **-s**, **-stats**, **-stats-report-frequency** - The frequency of SRT statistics collection, based on the number of packets.
```

If you perform tests on the public Internet, consider checking your firewall rules. The **SRT listener** must be reachable on the chosen UDP port. Same applies to routers using NAT. Please set a port forwarding rule with protocol UDP to the local IP address of the **SRT listener**.

> The direction of initiating the stream doesn't need to be the same as the stream direction. The sender of a stream can be a **SRT listener** or **SRT caller** as long as the receiving end uses the opposite mode. Typically you use the **SRT listener** on the end, which is easier to configure in terms of firewall/router setup. It also makes sense to leave the Sender in listener mode when trying to connect from various end points with possibly unknown IP addresses.

Reviewer: ```suggestion
The initiation of an SRT connection (handshake) is decoupled from the stream direction. The 
sender of a stream can be an **SRT listener** or an **SRT caller**, as long as the receiving end 
uses the opposite connection mode. Typically you use the **SRT listener** on the receiving end, 
since it is easier to configure in terms of firewall/router setup. It also makes sense to leave the 
Sender in listener mode when trying to connect from various end points with possibly 
unknown IP addresses.
```

Reviewer 2: I'd say it simpler:

The connection is in general bidirectional, so the stream sending direction is independent on how the connection was established and which roles are assigned to the particular endpoint. Typically you use the **SRT listener** on the end, on which it is easier to configure in terms of firewall/router setup, or you might want to use **rendezvous** mode on both sides in case when you need to punch through the firewall.

Also, it is a bad idea to mention the possibility of connecting multiple callers to one listener, as this is a documentation for `srt-live-transmit`, which does not support it. This application does allow to connect to one listener multiple times, but it handles only one connection at a time.
